modules/ROOT/pages/dataweave-troubleshoot.adoc

```diff
@@ -0,0 +1,105 @@
+= Troubleshooting
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: studio, anypoint, esb, transform, transformer, format, aggregate, rename, split, filter convert, xml, json, csv, pojo, java object, metadata, dataweave, data weave, datamapper, dwl, dfl, dw, output structure, input structure, map, mapping
+
+== Troubleshoot a Failing Script
+
+One challenge to reproducing an error is having the same inputs the script had
+when it executed, especially in production environments where inputs can
+change unexpectedly.
+
+In Mule 4.2.1, DataWeave introduced an experimental feature that enables you
+to dump the input context and the failing script into a folder so that you
+can track the failing script along with the data that makes the script fail.
+
+To use this feature:
+
+. Set the following system property to enable the dump feature: `-M-Dcom.mulesoft.dw.dump_files=true`.
+. [Optional] Specify the path in which to generate the dump files: `-M-Dcom.mulesoft.dw.dump_folder=<path_to_folder>`
++
+The default directory is `/tmp`.
+
+
+== Exception: org.mule.weave.v2.exception.UnsupportedTypeCoercionException
```

Comment:
Would it make sense to make this parallel to the failing script title by rewording it something like...:
```suggestion
== Troubleshooting Exceptions
```